plugin-packs/postcss-preset-env/README.md

```diff
@@ -159,6 +159,34 @@ postcss([
 - [Gulp](INSTALL.md#gulp)
 - [Grunt](INSTALL.md#grunt)
 
+### Position in your PostCSS config list
+
+[PostCSS Preset Env] only contains polyfills and fallbacks for standard CSS features.  
+[PostCSS Preset Env] is unable to handle non-standard features and syntactic sugar.
+
+If you also have PostCSS plugins for non-standard features and syntactic sugar,  
+you should place these first in the PostCSS config list.
+
+```js
+module.exports = {
+	plugins: [
+		"postcss-syntactic-sugar",
+		"postcss-non-standard",
```

Comment:
Could we mention `postcss-nested` maybe ? Because it's clearly a non-standard syntax. Or another if you have an example

Reply:
I didn't want to call out any plugin in particular because then it would seem as if only that specific plugin needs to be placed at a specific position.